docs/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Make workflows API RESTful (OSIDB-1716)
 - Minor change to enable perf tests to run in CI (OSIDB-2447)
+- Allow searching by CVE similarity (OSIDB-2482)
```

Comment:
This should be in the `### Added` part of the changelog (also, rebase is probably necessary).